Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ however, it has to be formatted properly to pass verification tests.
 
 ## [Unreleased]
 
+>__The minimum version requirement for the Input System package has been moved up to 2019.4 LTS.__
```

Comment:
Whats the reason for moving the min version to LTS?
Is it just because we want people to move to LTS?

Reply:
Pretty much. We haven't been able to backport native fixes to our min required version for quite some time now. With 2019.4 out, we can in good conscience move on to a newer version of Unity without forcing users onto 2020.x and at the same time give us a Unity version that we can keep backporting native fixes to and thus fully support. Plus we'll focus on the version that for now will likely have the majority of users. Last but not least, it'll give us a main target that will stay immovable for a good while.

Ah and... XR was already having a quasi-2019.3+ requirement so... time to move on :)